src/Cli.Tests/EndToEndTests.cs

```diff
@@ -9,48 +14,77 @@ namespace Cli.Tests;
 /// </summary>
 [TestClass]
 public class EndToEndTests
+    : VerifyBase
 {
-    /// <summary>
-    /// Setup the logger and test file for CLI
-    /// </summary>
-    [ClassInitialize]
-    public static void Setup(TestContext context)
+    private IFileSystem? _fileSystem;
+    private RuntimeConfigLoader? _runtimeConfigLoader;
+    private ILogger<Program>? _cliLogger;
+
+    [TestInitialize]
+    public void TestInitialize()
     {
-        if (!File.Exists(TEST_SCHEMA_FILE))
+        MockFileSystem fileSystem = new();
+
+        fileSystem.AddFile(
```

Comment:
can we move this initialize logic in TestHelper class and use it in all our test classes?

Reply:
Yep, I'll make a separate task for that as I think there's a bit of "general cleanup" we can do once this is merged to have some more shared code and things like that, but it's more "nice to have" than "must have".

Reply:
Created #1546 for a follow up task